templates/registration/includes/profile_portfolio.html

```diff
@@ -2,19 +2,22 @@
 
 <div class="investments">
     <h2>Portfolio</h2>
-    <p>Here are the artists that you have invested in:</p>
+    <div class="break">
+        <hr>
+    </div>
+    <h5>Here are the artists that you have invested in:</h5>
     {% include "artist/includes/investor_profile_artist_list.html" with artists=artists %}
     <div class="break">
         <hr>
     </div>
-    <h5 class="profile-total-header">TOTAL EARNED:</h5>
```

Comment:
Maybe this class should be renamed since it now doesn't seem to apply to headers.